care/emr/tests/test_file_upload_api.py

```
@@ -182,3 +202,131 @@ def test_cleanup_incomplete_file_uploads(self):
 
         with self.assertRaises(FileUpload.DoesNotExist):
             file_obj.refresh_from_db()
+
+    def test_service_request_file_upload_requires_authorization(self):
+        url = reverse("files-list")
+        with patch(
+            "care.emr.api.viewsets.file_upload.AuthorizationController.call"
+        ) as mock_auth:
+
+            def side_effect(method_name, *args, **kwargs):
```

Ruff (0.14.5)

Unused function argument: `args`

(ARG001)